js/form.js

```diff
@@ -0,0 +1,27 @@
+const adForm = document.querySelector('.ad-form');
+const adFieldsets = adForm.querySelectorAll('fieldset');
+const mapFormFilter = document.querySelector('.map__filters');
+const mapSelects = mapFormFilter.querySelectorAll('select');
+const mapFieldsets = mapFormFilter.querySelectorAll('fieldset');
+
+const changeStateElements = (elements, isDisabled) => {
+  isDisabled ?
+    elements.forEach((element) => element.setAttribute('disabled', 'disabled')) :
```

Comment:
Используй лучше обращение ко свойству `element.disabled` тогда можно убрать условие и заменить на 
```suggestion
    elements.forEach((element) => element.disabled = elementState;
```